Project/library/src/HistogramProcesser/HistogramCreator.cpp

```diff
@@ -4,6 +4,7 @@ using namespace std;
 
 void HistogramProcesser::getImageHistogram(int channel)
 {
+	if (channel == 3) return;
```

Comment:
To nie powinno być w getImageHistogram tylko w HistogramProcesser np. jako jeżeli opcja < centropy to getImageHistogram.